src/components/SiteHeader.vue

```diff
@@ -150,23 +117,127 @@
                   'flex',
                   'justify-center',
                   'items-center',
+                  'rounded-full',
+                  'min-w-[20px]',
+                  'min-h-[20px]',
+                  'ml-[-10px]',
+                  'mb-[-10px]',
+                  'px-[4px]',
+                  'py-[5px]',
+                  'pointer-events-none',
                   'bg-shade-gray',
                   { 'bg-danger': getNotificationCount },
-                  'rounded-full',
-                  'min-w-[24px]',
-                  'px-[8px]',
-                  'py-[4px]',
                 ]"
               >
-                <div class="text-white text-[10px]">
+                <div class="text-white text-[10px] leading-[1em]">
                   {{ formattedNotificationCount }}
                 </div>
               </div>
             </template>
           </MenuItem>
         </MenuList>
       </Dropdown>
+
+      {{ /* phone version */ }}
+      <ButtonV2
+        v-if="!getAddress"
+        class="laptop:hidden"
+        preset="plain"
+        @click="handleOpenSider"
+      >
+        <IconNav />
+      </ButtonV2>
+      <Identity
+        v-if="getAddress"
+        class="cursor-pointer laptop:!hidden ml-[12px]"
+        :avatar-url="walletUserAvatar"
+        :avatar-size="42"
+        :is-avatar-outlined="isWalletUserCivicLiker"
+        @click="handleOpenSider"
+      />
     </div>
+
+    <SiteMenuSider v-if="isShowSider" @close="isShowSider = false">
+      <ButtonV2
+        v-if="!getAddress"
+        class="w-full"
+        preset="secondary"
+        @click="connectWallet"
+        ><div class="flex gap-[12px]">
+          <IconLogin />
+          {{ $t('header_button_connect_to_wallet') }}
```

Comment:
Because the icon & text need to be centered, I used the default slot. 
Using prepend would create too much space
![截圖 2024-08-22 下午2 34 06](https://github.com/user-attachments/assets/41b2454e-817b-4991-8f8f-7c459e36aa43)